vidyut-prakriya/www/static/vidyut-prakriya-app.js

```diff
@@ -409,25 +442,38 @@ const App = () => ({
     },
 
     renderStepRule(rule) {
-        if (rule.source === "ashtadhyayi") {
-            let text = this.sutras[rule.code];
-            return text ? this.deva(text) : '';
-        } else if (rule.source === "varttika") {
-            let text = this.varttikas[rule.code];
-            text = this.deva(text || "");
-            if (text) {
-                return `<span class="text-green-500">${text}</span>`;
-            } else {
-                return '';
-            }
-        } else {
-            return "(missing)"
+        let text = '';
+        let annotated_text = 'missing';
+        switch (rule.source) {
+            case "ashtadhyayi":
+                text = this.devaNoSvara(this.sutras[rule.code] || "");
+                annotated_text = text;
```

Comment:
rather than `annotated_text = text; break`, can just do `return text`, likewise for the `case` statements below.